src/vs/workbench/contrib/themes/browser/themes.contribution.ts

```diff
@@ -693,3 +699,49 @@ MenuRegistry.appendMenuItem(ThemesSubMenu, {
 	order: 3
 });
 
+class DefaultThemeUpdatedNotificationContribution implements IWorkbenchContribution {
+
+	static STORAGE_KEY = 'themeUpdatedNotificationShown';
+
+	constructor(
+		@INotificationService private readonly _notificationService: INotificationService,
+		@IWorkbenchThemeService private readonly _workbenchThemeService: IWorkbenchThemeService,
+		@IStorageService private readonly _storageService: IStorageService,
+		@ICommandService private readonly _commandService: ICommandService,
+	) {
+		if (!this._workbenchThemeService.hasUpdatedDefaultThemes()) {
+			return;
+		}
+		if (_storageService.getBoolean(DefaultThemeUpdatedNotificationContribution.STORAGE_KEY, StorageScope.APPLICATION)) {
+			return;
+		}
+		setTimeout(() => {
+			this._showNotification();
+		}, 6000);
+	}
+
+	private async _showNotification(): Promise<void> {
+		this._storageService.store(DefaultThemeUpdatedNotificationContribution.STORAGE_KEY, true, StorageScope.APPLICATION, StorageTarget.USER);
+		await this._notificationService.notify({
+			id: 'themeUpdatedNotification',
+			severity: Severity.Info,
+			message: localize({ key: 'themeUpdatedNotification', comment: ['{0} is the name of the new default theme'] }, "VS Code now ships with a new default theme '{0}'. We hope you like it. If not, you can switch back to the old theme or try one of the many other color themes available.", this._workbenchThemeService.getColorTheme().label),
```

Comment:
@aeschli Wrong backtick.

Reply:
All is correct, right? No backtick needed here, localize does the substitution.

Reply:
Oops, you're right. Maybe this._workbenchThemeService.getColorTheme().label empty for some reason.